docs/source/en/model_doc/segformer.md

```diff
@@ -1,113 +1,107 @@
-SegFormer's results on the segmentation datasets like ADE20k, refer to the [paper](https://huggingface.co/papers/2105.15203).
+You can find all the original SegFormer checkpoints under the [Nvidia](https://huggingface.co/nvidia/models?search=segformer) organization.
 
-## Resources
+> [!TIP]
+> This model was contributed by [nielsr](https://huggingface.co/nielsr).
+>
+> Click on the SegFormer models in the right sidebar for more examples of how to apply SegFormer to different tasks like **semantic segmentation** and **image classification**.
+
+The example below demonstrates how to run **semantic segmentation** with [`Pipeline`] or the [`AutoModel`] class.
+
+<hfoptions id="usage">
+<hfoption id="Pipeline">
+
+```python
+from transformers import pipeline
+
+segmenter = pipeline("semantic-segmentation", model="nvidia/segformer-b0-finetuned-ade-512-512")
+image = "your_image.png"
+outputs = segmenter(image)
+```
+
+</hfoption>
+<hfoption id="AutoModel">
 
-A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with SegFormer.
+```python
+from transformers import SegformerImageProcessor, SegformerForSemanticSegmentation
+from PIL import Image
+import requests
 
-<PipelineTag pipeline="image-classification"/>
+url = "http://images.cocodataset.org/val2017/000000039769.jpg"
+image = Image.open(requests.get(url, stream=True).raw)
 
-- [`SegformerForImageClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/image-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/image_classification.ipynb).
-- [Image classification task guide](../tasks/image_classification)
+processor = SegformerImageProcessor.from_pretrained("nvidia/segformer-b0-finetuned-ade-512-512")
+model = SegformerForSemanticSegmentation.from_pretrained("nvidia/segformer-b0-finetuned-ade-512-512")
 
-Semantic segmentation:
+inputs = processor(images=image, return_tensors="pt")
+outputs = model(**inputs)
+logits = outputs.logits  # shape [batch, num_labels, height, width]
+```
 
-- [`SegformerForSemanticSegmentation`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/semantic-segmentation).
-- A blog on fine-tuning SegFormer on a custom dataset can be found [here](https://huggingface.co/blog/fine-tune-segformer).
-- More demo notebooks on SegFormer (both inference + fine-tuning on a custom dataset) can be found [here](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/SegFormer).
-- [Semantic segmentation task guide](../tasks/semantic_segmentation)
+</hfoption>
+
+</hfoptions>
+
+Quantization reduces the memory burden of large models by representing the weights in a lower precision. Refer to the [Quantization](../quantization/overview) overview for more available quantization backends.
```

Comment:
The model isn't very big so no need to have a quantization example

Reply:
Gotcha, removed it